models/api.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection")
+
+
+exports.retrieveTopics = async () => {
+    const query = 'SELECT * FROM topics;'
+    const result = await db.query(query)
+    console.log("getting result ", result)
```

Comment:
Do not leave console logs in complete and working code. This tells me you don't have confidence in your tests and would rather log.